checks/nongplsites.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Checks if the theme includes resoruces from websites that does not use a GPL compatible license.
+ */
+class NonGPLCheck implements themecheck {
+	protected $error = array();
+
+	function check( $php_files, $css_files, $other_files ) {
+
+		$ret  = true;
+		$code = implode( ' ', $other_files ); // The references are usually in the readme.
+
+		checkcount();
+
+		$link_list = array(
+			'unsplash'            => 'https://unsplash.com/license',
```

Comment:
Maybe we should remove protocol, so if `http` or `https` is uses this would catch it, no?

Reply:
The url is not used in the check, it is only so that the user can go there to read the license.